fullcontact/api/company_api.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module serves the class for making
+FullContact Company Enrich and Search
+API requests.
+"""
+
+from concurrent.futures import Future
+
+from .base.enrich_base import EnrichBase
+from ..response.company_response import CompanyEnrichResponse, CompanySearchResponse
+from ..schema.company_schema import CompanyEnrichSchema, CompanySearchSchema
+
+
+class CompanyApi(EnrichBase):
+    r"""
+    Class that provides methods to perform
+    Company Enrich and Search operations.
+    """
+    _enrich_endpoint = "company.enrich/"
```

Comment:
This is *super minor* but should we remove the trailing slash to match the docs?

Reply:
Sure, let us stay consistent! I'll remove that.

Reply:
Removed.